### PR TITLE
Revert "[agentbeat] Fix agentbeat capabilities"

### PR DESCRIPTION
**Pull request by blakerouse**
Reverts elastic/beats#40466 as the change in this PR https://github.com/elastic/elastic-agent/pull/5290 removes the need for the agentbeat itself to promote its own capabilites.

### CODE REVIEW

**Review by cmacknz — APPROVED** [review]

File: x-pack/agentbeat/main.go [hunk 1]
```diff
@@ -70,12 +70,8 @@ func prepareCommand(rootCmd *cmd.BeatsRootCmd) *cobra.Command {
 		// filename, as all the beats set this in the initialization.
 		err := cfgfile.ChangeDefaultCfgfileFlag(rootCmd.Use)
 		if err != nil {
-			panic(fmt.Errorf("failed to set default config file path: %w", err))
+			panic(fmt.Errorf("failed to set default config file path: %v", err))
```
Comment (mauri870) [inline x-pack/agentbeat/main.go:73]: Should this be a %w instead?

Comment (belimawr) [reply x-pack/agentbeat/main.go:73]: Even though I believe nothing will ever try to recover from this panic, I believe it's good to make the linter happy.

Comment (blakerouse) [reply x-pack/agentbeat/main.go]: Fixed.

**Review by mauri870 — APPROVED** [review]

**Comment by VihasMakwana** [comment]
@blakerouse I think we should backport this revert PR, because my [original PR](https://github.com/elastic/beats/pull/40489) was also backported. 
cc: @pierrehilbert

**Pull request merged**